source/ui/x11/X11Window.ooc

```diff
@@ -52,8 +64,34 @@ X11Window: class extends NativeWindow {
 	}
 	resize: func (size: IntSize2D) { XResizeWindow(this display, this backend, size width, size height) }
 	free: override func {
+		if (this _xImage)
+			XDestroyImage(this _xImage)
+		XFreeGC(this display, this _x11GraphicsContext)
+		XDestroyWindow(this display, this backend)
 		XCloseDisplay(this display)
 		super()
 	}
+	draw: func (image: RasterYuv420Semiplanar) {
```

Comment:
Can `operator[]` be declared as virtual in `RasterImage` or `RasterPacked` ? Will it work if we reimplement it in derived classes ?
Another option is to have a `getColor: abstract func (x,y: Int) -> ColorBgr` somewhere in the image hierarchy.
Anyway let's fix that in another PR, this one is good enough to run noGpu on linux.